index.html

```diff
@@ -18,169 +18,331 @@
+								></a>
+							</li>
+						</ul>
+					</div>
 
-				<button class="resume" type="submit">Get my resume</button>
+					<button class="btn" type="submit">Get my resume</button>
 
-				<ul class="skills">
-					<li class="skills-item">
-						<h3>
-							Languages <i class="fa-solid fa-angle-down"></i>
-						</h3>
-						<div class="skills-type">
-							<span>
-								<img src="images/html.png" alt="" /> HTML</span
-							>
-							<span><img src="images/css.png" alt="" /> CSS</span>
-							<span
-								><img src="images/js.png" alt="" />
-								JavaScript</span
-							>
-						</div>
-					</li>
+					<ul>
+						<li class="dropdown-skill">
+							<div class="dropdown-headers">
+								<h3 class="dropdown-title">Languages</h3>
+								<button type="button" class="icon_btn_dropdown">
+									<i class="fa-solid fa-angle-down"></i>
+								</button>
+							</div>
+
+							<div class="dropdown-skill-item">
+								<ul class="vertical-list">
+									<li class="mini-card">
+										<img
+											src="images/html.png"
+											alt="html-log"
+										/>
+										<h3 class="mini-card-title">HTML</h3>
+									</li>
+									<li class="mini-card">
+										<img
+											src="images/css.png"
+											alt="css-logo"
+										/>
+										<h3 class="mini-card-title">CSS</h3>
+									</li>
+									<li class="mini-card">
+										<img
+											src="images/js.png"
+											alt="javaScript-logo"
+										/>
+										<h3 class="mini-card-title">
+											JavaScript
+										</h3>
+									</li>
+								</ul>
+							</div>
+						</li>
+						<li class="dropdown-skill">
+							<div class="dropdown-headers">
+								<h3 class="dropdown-title">Frameworks</h3>
+								<button type="button" class="icon_btn_dropdown">
+									<i class="fa-solid fa-angle-right"></i>
+								</button>
+							</div>
+						</li>
+						<li class="dropdown-skill">
+							<div class="dropdown-headers">
+								<h3 class="dropdown-title">Skills</h3>
+								<button type="button" class="icon_btn_dropdown">
+									<i class="fa-solid fa-angle-right"></i>
+								</button>
+							</div>
+						</li>
+					</ul>
+				</div>
+			</section>
+
+			<section id="contact" class="contact">
+				<div class="section-title">
+					<h2 class="title">Contact me</h2>
+					<p class="text_contact">
+						If you have an application you are interested in
+						developing, a feature that you need to built a project
+						that needs coding; I would love to help with it.
+					</p>
+					<div>
+						<form
+							action="https://formspree.io/f/xgedpywn"
+							method="post"
+							id="form"
+						>
+							<div class="info">
+								<div class="form-group">
+									<label for="name">Name:</label>
+									<input
+										type="text"
+										id="name"
+										name="user-name"
+										placeholder="Full name"
+										required
+									/>
+								</div>
```

Comment:
- [ ] [[OPTIONAL] Kindly add the `maxlength="30"` attribute, this will enable you restrict the number of characters the Name input can take